web-app/src/hoc/Layout/header.jsx

```diff
@@ -36,98 +32,75 @@ const Header = ({ history, isAuthenticated }) => {
     });
   };
 
-  const inAdminSection = (match, location) => {
-    const { pathname } = location;
-    if (pathname.indexOf('admin') != -1) {
-      return true;
+  const creatTopLevelNavLinks = headerLink => {
```

Comment:
You've missed an 'e' in the const name here, I would fix this up in case someone trips on it later.